src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -0,0 +1,175 @@
+package reposense.parser;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Optional;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import reposense.exception.ParseException;
+import reposense.frontend.CliArguments;
+import reposense.util.TestUtil;
+
+public class ArgsParserTest {
+
+    private static final String INPUT_DELIMITER = " ";
+    private static final String PROJECT_DIRECTORY = System.getProperty("user.dir");
```

Comment:
Not very good, what if I try to run the test from some other directory?

Would it be safer to do it somewhat similarly like functional test?https://github.com/reposense/RepoSense/blob/b7bab4cea243aca7ea857d49d7c3935ee1e276c0/src/functional/java/reposense/Entry.java#L43

Reply:
I have tried implementing with ```ClassLoader``` to get **project base directory** but JUnit tests only passed in IntelliJ and failed in gradlew and travis. 

On further investigations, it seems like ```gradlew``` can only be run in the **project base directory** so using ```System.getProperty("user.dir")``` makes sense.

Reply:
Hmm okay, but place test files within the test folder (maybe like functional, create its own resource folder) to prevent any modification not relevant to the test.